source/hassio/zwave.markdown

```diff
@@ -42,6 +42,14 @@ zha:
   database_path: /config/zigbee.db
 ```
 
+### {% linkable_title INTEL NUC %}
+
+If you have INTEL NUC and host Ubuntu, on some system's the Z-Wave Aeotec stick is not starting up correctly - no Z-Wave nodes on Home Assistant start. Then you have to disable ModemManager:
+
+```bash
+ubuntu$ systemctl disable ModemManager.service
```

Comment:
Change `ubuntu$` to `$`

Reply:
As a matter of fact, leave out `ubuntu$` entirely.
Markdown standards only suggest on using `$` prefixing when actual output is shown. In case only commands are giving, the `$` should be left out.

Reply:
Addressed